source/administration/server-side-encryption.rst

```diff
@@ -51,10 +51,12 @@ supports the following encryption strategies:
       Deployment, see :ref:`minio-encryption-sse-kms-quickstart`. For
       production MinIO deployments, use one of the following guides:
 
-      - :ref:`AWS SecretsManager <minio-sse-aws>`
-      - :ref:`Google Cloud SecretManager <minio-sse-gcp>`
-      - :ref:`Azure Key Vault <minio-sse-azure>`
-      - :ref:`Hashicorp KeyVault <minio-sse-vault>`
+      - :kes-docs:`Azure Key Vault <integrations/azure-keyvault/>`
+      - :kes-docs:`AWS SecretsManager <integrations/aws-secrets-manager/>`
+      - :kes-docs:`Fortanix SDKMS <integrations/fortanix-sdkms/>`
+      - :kes-docs:`Google Cloud SecretManager <integrations/google-cloud-secret-manager/>`
+      - :kes-docs:`Hashicorp KeyVault <integrations/hashicorp-vault-keystore/>`
```

Comment:
```suggestion
      - :kes-docs:`Hashicorp Vault <integrations/hashicorp-vault-keystore/>`
```

Reply:
The product name typos are mainly fix once, copy elsewhere so I didn't note each one.

Reply:
Fixed throughout, thanks for the catch.